packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/AddBankYapily/SelectBank/template.success.tsx

```diff
@@ -30,29 +31,35 @@ const Success = (props: Props) => {
   }
 
   return (
-    <BankWrapper>
-      <ModalNavWithBackArrow {...props}>
+    <FlyoutContainer>
+      <FlyoutHeader
+        mode='back'
+        data-e2e='OpenBankingSelectBankBackButton'
+        onClick={props.handleClose}
+      >
         <FormattedMessage id='copy.find_your_bank' defaultMessage='Find Your Bank' />
-      </ModalNavWithBackArrow>
-      <BankSearchWrapper>
-        <BankSearchInput onChange={simpleSearch} placeholder='Search' type='text' />
-        <BankSearchIcon />
-      </BankSearchWrapper>
-      {banks.map((bank) => {
-        return (
-          <SimpleBankRow
-            key={bank.id}
-            institution={bank}
-            onClick={() => {
-              props.setYapilyBankId(bank.id)
-              props.brokerageActions.setAddBankStep({
-                addBankStep: AddBankStepType.ADD_BANK_AUTHORIZE
-              })
-            }}
-          />
-        )
-      })}
-    </BankWrapper>
+      </FlyoutHeader>
+      <FlyoutContent mode='top'>
+        <BankSearchWrapper>
+          <BankSearchInput onChange={simpleSearch} placeholder='Search' type='text' />
+          <BankSearchIcon />
+        </BankSearchWrapper>
+        {banks.map((bank) => {
+          return (
```

Comment:
just a silly one you can remove return and `{}` and just use component as returned statement :)

Reply:
Good spot. Done.